### ISSUE TITLE
Feature Request: background colour

### ISSUE THREAD

**Issue by jerry1970** [vektah/CodeGlance#206]
A background colour setting would be nice, to make a distinction between the editor and the codeglance area.

By the way, the "Viewport color" setting does not seem to do anything at all. I have no idea what this does. (I use the Darcula theme, by the way.)

**Comment by Leo-ZihaoLiu** [comment]
Agreed. After setting a picture as background of editor, viewport's color has been affected and cannot be fixed.
